src/calendar-heatmap.tsx

```diff
@@ -738,7 +738,7 @@ export class CalendarHeatmap extends Component<
       );
 
     // Add month data items to the overview
-    this.items.selectAll('.item-block-month').exit().remove();
```

Comment:
with exit(), while pressing back button and coming back to month overview it is not rendering the element.